Modules/rat_rescue.py

```diff
@@ -82,6 +70,10 @@ def __init__(self, uuid: UUID,
             irc_nickname (str): clients IRC nickname, may deffer from their
                 commander name.
             rats (list): identified (Rat)s assigned to rescue.
+            rescue_uuid(UUID): Internal uuid used for hashing
```

Comment:
Perhaps the constructor arg and property name could be changed to clarify its purpose. `rescue_uuid` could refer to the api id as well. Something along the lines of `internal_id` of `hashing_id`.